### ISSUE TITLE
Modelless window for Lookup tools

### ISSUE THREAD

**Issue by jmcouffin** [jeremytammik/RevitLookup#92]
Hi @jeremytammik, 
while not able to code it myself, especially not in C#, 
a nice feature would be to get the windows of the tool in modelless mode. Allowing us to interact with the model directly.
Cheers

**Comment by jeremytammik** [comment]
Hi @jmcouffin , I totally agree, that sounds like a super cool and useful idea. It might require quite a bit of work. Although, if it is limited to the currently active document at all times, it might not be all that much. Probably it all depends on picking a clever simple minimalistic architectural approach. Let's keep this issue open then, and see whether someone will volunteer to pick it up some day.

**Comment by NeVeSpl** [comment]
I loved the idea, so I had to implement it, #93. I hope that you will enjoy modeless RevitLookup as much as I do.

**Comment by jeremytammik** [comment]
yup, indeed. i integrated and compiled #93 + #94. have not tested yet, though.
